updater.ps1

```diff
@@ -24,7 +24,7 @@ function FindAllRepos {
         [string] $PAT
     )
 
-    $url = "https://api.github.com/orgs/$orgName/repos"
+    $url = "https://api.github.com/orgs/$orgName/repos?per_page=100"
```

Comment:
does this limit the output to only 100 repos? What if the org has > 100? Should we use pagination headers here (and handle them accordingly), to be future proof?

Reply:
Yes, That is right. It was by default 30 and we were never checking for updates for some repos already. 
This is only a temporary fix and will implement pagination in another PR.